### PR TITLE
DF-1214 Threshold for UserEventIntegration set up

### PR DESCRIPTION
**Pull request by dvoracek-martin**

### CODE REVIEW

File: jobroom-gateway-app/src/main/java/ch/admin/seco/jobroom/service/impl/messaging/UserEventIntegrationChannels.java [hunk 1]
```diff
@@ -33,7 +37,7 @@ public QueueChannel queueChannel() {
 
             @Override
             public int threshold() {
-                return 0;
+                return userEventIntegrationThreshold;
```
Comment (birom) [inline jobroom-gateway-app/src/main/java/ch/admin/seco/jobroom/service/impl/messaging/UserEventIntegrationChannels.java]: Why do not we provide a default implementation in the ``alv-spring-integration-starter``?

Comment (pavax) [reply jobroom-gateway-app/src/main/java/ch/admin/seco/jobroom/service/impl/messaging/UserEventIntegrationChannels.java]: very good idea

Comment (dvoracek-martin) [reply jobroom-gateway-app/src/main/java/ch/admin/seco/jobroom/service/impl/messaging/UserEventIntegrationChannels.java]: Adjusted to use the default value from the shared library

File: jobroom-gateway-app/src/main/resources/config/application-dev.yml [hunk 2]
```diff
@@ -186,3 +186,7 @@ security:
                 location: classpath:uid/jks/uid-client-truststore.jks
                 password: MUST_BE_PROVIDED_BY_THE_CONFIG_SERVER
 
+user:
```
Comment (birom) [inline jobroom-gateway-app/src/main/resources/config/application-dev.yml]: We do not need this any more. Please remove it.

File: jobroom-gateway-app/src/main/resources/config/application.yml [hunk 3]
```diff
@@ -210,3 +210,7 @@ security:
     user:
         logout_url: '/saml/logout'
         redirect-base-url:  ${jhipster.mail.base-url}
+user:
```
Comment (birom) [inline jobroom-gateway-app/src/main/resources/config/application.yml]: We do not need this any more. Please remove it.

File: jobroom-gateway-app/src/main/java/ch/admin/seco/jobroom/service/impl/messaging/UserEventIntegrationChannels.java [hunk 4]
```diff
@@ -14,6 +15,9 @@
 
     private final ChannelMessageStore channelMessageStore;
 
+    @Value("${user.event.integration.threshold:20}")
+    private int userEventIntegrationThreshold;
```
Comment (birom) [inline jobroom-gateway-app/src/main/java/ch/admin/seco/jobroom/service/impl/messaging/UserEventIntegrationChannels.java]: It is not used => remove it

**Pull request merged**